src/Altinn.AccessManagement.Core/Helpers/PathHelper.cs

```diff
@@ -0,0 +1,35 @@
+namespace Altinn.AccessManagement.Core.Helpers
+{
+    /// <summary>
+    /// Helper class for Path manipulation and checks
+    /// </summary>
+    public static class PathHelper
```

Comment:
is this util used in the user profile scenario?

Reply:
Not used, I'll remove it